src/applications/check-in/locales/en/translation.json

```diff
@@ -147,5 +147,7 @@
   "wife": "Wife",
   "relationship-is-required": "Relationship is required",
   "niece-nephew": "Niece/Nephew",
-  "add": "Add"
+  "add": "Add",
+  "in-en": "in english",
+  "in-es": "in spanish"
```

Comment:
this is a minor thing but I don't think we'll ever link to an external  spanish-language page from the english version of our app. We might want to take `in-es` out to avoid confusion?

Reply:
I think my initial though was to have it in there as a fallback, but agree that leaving it out is probably the better choice so we don't use it somewhere by mistake. These have been removed from the PR.